bip-ccv.mediawiki

```diff
@@ -0,0 +1,420 @@
+<pre>
+  BIP: TBD
+  Layer: Consensus (soft fork)
+  Title: OP_CHECKCONTRACTVERIFY
+  Author: Salvatore Ingala <salvatoshi@protonmail.com>
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-TBD
+  Status: Draft
+  Type: Standards Track
+  Created: TBD
+  License: BSD-3-Clause
+  Post-History: 2022-11-08: https://gnusha.org/pi/bitcoindev/CAMhCMoH9uZPeAE_2tWH6rf0RndqV+ypjbNzazpFwFnLUpPsZ7g@mail.gmail.com
+                2023-07-30: https://gnusha.org/pi/bitcoindev/CAMhCMoFYF+9NL1sqKfn=ma3C_mfQv7mj2fqbqO5WXVwd6vyhLw@mail.gmail.com
+</pre>
+
+
+== Introduction ==
```

Comment:
Nit: Missing "Abstract" section.

Reply:
9bccf545930f9d4b8211f11d5bd93eac27b75695: I renamed the `Introduction` to `Abstract`.